tool/cli.js

```diff
@@ -219,6 +240,40 @@ program
       const url = `http://localhost:${PORT}${buildURL(locale, slug)}`;
       await open(url);
     })
+  )
+
+  .command("flaws", "Find (and fix) flaws in a document")
+  .argument("<slug>", "Slug of the document in question")
+  .argument("[locale]", "Locale", {
+    default: DEFAULT_LOCALE,
+    validator: [...VALID_LOCALES.values()],
+  })
+  .option("-y, --yes", "Assume yes", { default: false })
+  .action(
+    tryOrExit(async ({ args, options }) => {
+      const { slug, locale } = args;
+      const { yes } = options;
+      const document = Document.findByURL(buildURL(locale, slug));
+      const { doc } = await buildDocument(document, {
+        fixFlaws: true,
+        fixFlawsDryRun: true,
+      });
+
+      const flaws = Object.values(doc.flaws)
+        .map((a) => a.filter((f) => f.fixable).length || 0)
+        .reduce((a, b) => a + b);
+      const { run } = yes
+        ? true
+        : await prompts({
+            type: "confirm",
+            message: `Proceed fixing ${flaws} flaws?`,
+            name: "run",
+            initial: true,
+          });
```

Comment:
This is broken when `-y, --yes` is used on the command line:
```sh
rjohnson-44864:deployer rjohnson$(maws_profile)$ yarn tool flaws Web --yes
yarn run v1.22.5
$ node tool/cli.js flaws Web --yes
Would fix (link1) broken_link /en-US/docs/Web/Apps to /en-US/docs/Web/Progressive_web_apps
Would fix (link4) broken_link /en-US/docs/SVG to /en-US/docs/Web/SVG
Would modify "/Users/rjohnson/repos/content/files/en-us/web/index.html" from fixable flaws.
✨  Done in 1.30s.
```

Maybe something like:
```suggestion
      const run = yes
        ? true
        : await prompts({
            type: "confirm",
            message: `Proceed fixing ${flaws} flaws?`,
            name: "run",
            initial: true,
          }).run;
```